cases/forms/reporting.py

```diff
@@ -91,7 +91,7 @@ class PostcodeForm(StepForm):
     def clean_postcode(self):
         pc = self.cleaned_data["postcode"]
         addresses = cobrand.api.addresses_for_postcode(pc)
-        if "error" in addresses:
+        if "error" in addresses or not len(addresses["addresses"]):
```

Comment:
I suppose for extra defensiveness this could be `len(addresses.get("addresses", []))`, but you may have a better idea than I do whether the API always includes that key. Fine as is though :)

Reply:
you're right, I shouldn't trust anything